src/VisualStudio/CSharp/Impl/CSharpVSResources.resx

```diff
@@ -513,4 +513,7 @@
   <data name="When_possible" xml:space="preserve">
     <value>When possible</value>
   </data>
+  <data name="Report_invalid_placeholders_in_string_dot_format_expressions" xml:space="preserve">
+    <value>Report invalid placeholders in 'string.Format' expressions</value>
```

Comment:
i think i would prefer: ... in 'string.Format' calls

instead of saying "expressions".